docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -76,6 +84,13 @@ <h5>
               Files are linked in <code>/tmp</code> using symbolic links back to original locations.
             </td>
           </tr>
+           <tr>
```

Comment:
would it be possible to see this html and how it looks ? using Github docs.

Reply:
https://tst-mswartz.github.io/earthenterprise/geedocs/5.3.0/answer/7160007.html

Reply:
we need to update copyrights year(s) to: 2018-2019 for the release notes